docs/en/userguide/02_Administrator_manual/02_Resetting_accounts.md

```diff
@@ -0,0 +1,21 @@
+title: Resetting accounts
+summary: How to reset a user's account if they lose access to it
+
+# Resetting accounts
+
+As an administrator, you can assist other users that have been locked out of
+their accounts by sending them an Account Reset request via the CMS.
```

Comment:
As mentioned before, I'm not sold on referring to "an Account Reset request" - something like "an email to reset their account" might be better